### ISSUE TITLE
Shortcut - App does not redirect to group chat when splitting bill via Shortcut

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#39300]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.58-1 
**Reproducible in staging?:**Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** [N/A](https://expensify.testrail.io/index.php?/tests/view/4463265)
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to FAB > Start chat
3. Create a group chat
4. Send a message and split bill in the group chat
5. Navigate to a different chat other than the group chat
6. Go to FAB > Split bill under Shortcut
7. Split another bill

## Expected Result:
After splitting bill via Shortcut, app will open the group chat

## Actual Result:
After splitting bill via Shortcut, app does not open the group chat

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details>  

https://github.com/Expensify/App/assets/78819774/b16eb3b0-c043-430e-a4ef-6cc37d555ef9

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
@joekaufmanexpensify  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by lanitochka17** [comment]
We think that this bug might be related to #vip-split

**Comment by abzokhattab** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
App does not redirect to chat report when splitting bill via Shortcut
### What is the root cause of that problem?

we dont navigate to the chat report when dismissing the create split modal here:
https://github.com/Expensify/App/blob/cbb1a7bf799296271c72d9a6bc3a8411b7dc59ea/src/libs/actions/IOU.ts#L2683

 
### What changes do you think we should make in order to solve the problem?


we need to pass the `existingSplitChatReportID` to the dismiss modal function.
```js
    Navigation.dismissModal(existingSplitChatReportID);
```

**Comment by joekaufmanexpensify** [comment]
will triage today

**Comment by joekaufmanexpensify** [comment]
Yeah this feels like a bug imo. cc @Gonals as I know you're leading QAB

**Comment by abzokhattab** [comment]
[The PR](https://github.com/Expensify/App/pull/39577) is ready! Please let me know if you have any comments

**Comment by joekaufmanexpensify** [comment]
Great. TY!

**Comment by joekaufmanexpensify** [comment]
PR is on prod. Automation wasn't working, but payment is due in 2 days (2024-04-17)

**Comment by joekaufmanexpensify** [comment]
All set to issue payment here. We need to pay: 

- @abzokhattab $500 for PR (paid via upwork). 
- @akinwale $500 for C+ (paid via upwork). 

Noting that both payments are $500 here since this issue was opened prior to the baseline being set at $250.

**Comment by joekaufmanexpensify** [comment]
@abzokhattab $500 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
@akinwale $500 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
Upwork job closed.

**Comment by joekaufmanexpensify** [comment]
All set, thanks everyone!
